app/views/post.php

```diff
@@ -17,20 +19,19 @@
       <a href="./">back to list</a>
     </div>
     
-    <h1 data-wio-id=<?= $post->getId() ?>>
-      <?= $post->getText('post.title') ? $post->getText('post.title') : "Untitled" ?>
+    <h1 data-wio-id=<?= $post->id ?>>
+      <?= RichText::asHtml($post->data->title, $prismic->linkResolver) ?>
```

Comment:
You have the title text defined above as $title. No need to output this here as html.